brickstrap-utils.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
```

Comment:
Not strictly required, no. I kept it as a handy reminder that it is bash and not sh, though.